Sources/Webservice/Requests/RequestBehavior.swift

```diff
@@ -74,4 +75,8 @@ private struct CompoundRequestBehavior: RequestBehavior {
             $0.after(completion: result)
         }
     }
+
+    func allowCompletion(data: Data?, response: URLResponse?, error: Error?) -> Bool {
+        return !behaviors.map { $0.allowCompletion(data: data, response: response, error: error) }.contains(false)
```

Comment:
this can be more efficient, if any return false there is no point processing more

Reply:
Not sure if this is valid. We don't know what it is in the other behaviours and whether they know about this implementation detail. We also didn't discuss if one behaviour returns true and one returns false, what should we do? E.g. a logging behaviour.